gatsby-theme-mds/src/components/Rules/Rules.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { styled } from '@storybook/theming';
+import { Row, Column } from '@innovaccer/design-system';
+
+const StyledDiv = styled.div(({ BgColor }) => ({
+  backgroundColor: BgColor
+}));
+
+const Rules = ({ children, isLandscapeMode, BgColor }) => {
+
+  return (
+    <StyledDiv
+      BgColor={BgColor}
+    >
+      {
+        isLandscapeMode ?
```

Comment:
By default is should be in landscape mode , there should be a flag `portraitMode` with default to false value.